### PR TITLE
Ai branch

### PR DESCRIPTION
**Pull request by Tomerd-afk**
Added in a shotgun 
Revamped Prefabs for CYBORG and CYBORG SHOTGUN 
Added HP and Killability to AI so they can die and respawn over and over and over and over 

With my current build I doubt someone could last 30 seconds unless they run!

### CODE REVIEW

File: Assets/Scripts/Bullets/BasicRifleBullet.cs [hunk 1]
```diff
@@ -21,5 +21,10 @@ private void OnTriggerEnter(Collider other)
             tracerHealth.TakeDamage(damageDealt);
             //Debug.Log("Hit Player!");
         }
+
+        if (other.gameObject.tag == "Enemy")
```
Comment (toryjleo) [inline Assets/Scripts/Bullets/BasicRifleBullet.cs:25]: What happens in this case?

File: Assets/Scripts/Health.cs [hunk 2]
```diff
@@ -7,18 +7,23 @@ public class Health : MonoBehaviour
 {
     private float _hitPoints;
 
+
+    //WFT is this? a getter?
```
Comment (toryjleo) [inline Assets/Scripts/Health.cs:11]: C# properties. Yes, a getter: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties

File: Assets/Scripts/Health.cs [hunk 3]
```diff
@@ -7,18 +7,23 @@ public class Health : MonoBehaviour
...
 
+
+    //WFT is this? a getter? 
     public float HitPoints
     {
         get => _hitPoints;
     }
 
+
+    //Whem do I call this method and why? Do I have to call it every time?
```
Comment (toryjleo) [inline Assets/Scripts/Health.cs:18]: Fair point

File: Assets/Scripts/Bullets/BasicRifleBullet.cs (hunk 1 above)
Comment (Tomerd-afk) [reply Assets/Scripts/Bullets/BasicRifleBullet.cs:25]: Leaving it out for now

File: Assets/Scripts/Health.cs (hunk 2 above)
Comment (Tomerd-afk) [reply Assets/Scripts/Health.cs:11]: Sweet

**Pull request merged**